### PR TITLE
Jpk/invoice links

### PR DESCRIPTION
**Pull request by jpknoll**

### CODE REVIEW

**Comment by jpknoll** [comment]
We might consider adding the actual id into the link to prevent duplicate random ids

File: src/Payments.Mvc/Controllers/InvoicesController.cs [hunk 1]
```diff
@@ -163,6 +170,9 @@ public IActionResult Create()
                 return BadRequest("Invoice already sent.");
             }
 
+            // setup random 10 character key link id
+            invoice.LinkId = InvoiceKeyHelper.GetUniqueKey(10);
```
Comment (jSylvestre) [inline src/Payments.Mvc/Controllers/InvoicesController.cs]: @jpknoll Why not just a guid?

Comment (jpknoll) [reply src/Payments.Mvc/Controllers/InvoicesController.cs]: GUIDs are hard to speak over the phone. I'd actually consider using a /smaller/ key than 10, like 6-8 characters.

Comment (jSylvestre) [reply src/Payments.Mvc/Controllers/InvoicesController.cs]: @jpknoll Maybe throw in a check to see if the link already exists? Not sure what the collision chance is, but there isn't anything on the DB right now to say it has to be unique.

**Pull request merged**